.dockerignore

```diff
@@ -79,5 +79,5 @@ cypress
 # Vido
 **/*.stories.ts
 **/*.story.vue
-vidos.config.ts
+# vidos.config.ts
```

Comment:
No, we really want to ignore it.
It is local config file.

Reply:
Same thing, as it is generated from vidos.config.ts

Reply:
After thinking, I don't understand why we ignore it from Docker being into the docker image build BUT we add it as a volume in docker-compose ?

Reply:
We need `vidos.config.ts` if we want to execute `yarn build-config` which produces `vidos-config.json` file in whatever machine.